gf2.cpp

```diff
@@ -220,6 +220,7 @@ struct Breakpoint {
 	int watchpoint;
 	int hit;
 	bool enabled;
+	char condition[1024];
```

Comment:
This could probably be a bit smaller, say `128` bytes? And you can use the `MapShort::Hash` function to get a hash, for exact comparisons. (This function should be marked as `static`.)

Reply:
i don't understand. can you be more clear, do you want me to save the hash of the condition in condition[] instead of the raw string ? that can't be, because we need the raw string to display it in the condition column in the Breakpoints window (unless there's  a way to get the string back from the hash?). Or just use MapShort::Hash() in the comparison in the line 853 ?
And i don't really understand why do we need to use the hash, i don't see why strcmp() isn't enough ?

Reply:
Yes, the idea is the hash is also saved in the `Breakpoint` structure. Everything else looks good otherwise!